src/explore-education-statistics-frontend/src/components/PrintThisPage.tsx

```diff
@@ -18,7 +28,10 @@ const PrintThisPage = ({ analytics, ...props }: AnalyticProps) => {
   return (
     <div
       className={classNames(
-        'govuk-!-margin-top-6',
+        {
+          'govuk-!-margin-top-6': !noMargin,
+          [styles.alignCentre]: alignCentre,
```

Comment:
Is there not a `govuk` util class for `text-align: center`? If not, we should have one under the `dfe` namespace really.

Reply:
added in dfe-align classes to common